04_HowTos/FileService/How_to_use_the_File_Services.html

```diff
@@ -13092,31 +13093,76 @@
-<p><em>Revised:  Jan 03, 2019</em></p>
-<p>Files in the virtual storage are usually identified via the prefix "<em>vos://</em>". This shorthand identifier is resolved to a user's home directory of the storage space in the service.</p>
-<p>If the "<em>vos://</em>" prefix is instead the name of another user (e.g. "<em>geychaner://</em>", and the remainder of the path grants public or group read/write access, then the other user's spaces will be accessed. Most user spaces have a "<em>/public</em>" directory to facilitate file sharing (e.g. '<em>geychaner://public/foo.fits</em>' will access the '<em>foo.fits</em>' file from user '<em>geychaner</em>'). Users can make any file (or directory) public by moving it to (or creating a link in) their "/public" directory.</p>
-<p><em>Public file services</em> are specially created areas where all files are world-readable, and are used for serving files from Data Lab datasets.</p>
+<h1 id="How-to-Use-the-Data-Lab-Public-File-Services">How to Use the Data Lab Public File Services<a class="anchor-link" href="#How-to-Use-the-Data-Lab-Public-File-Services">&#182;</a></h1><p><em>Revised:  Sept 01, 2020</em></p>
 
+</div>
+</div>
+</div>
+<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
+</div><div class="inner_cell">
+<div class="text_cell_render border-box-sizing rendered_html">
+<h3 id="Table-of-contents">Table of contents<a class="anchor-link" href="#Table-of-contents">&#182;</a></h3><ul>
+<li><a href="#summary">Summary</a></li>
+<li><a href="#attribution">Disclaimer &amp; Attribution</a></li>
+<li><a href="#import">Imports &amp; setup</a></li>
+<li><a href="#listspace">Listing another user's public file space</a></li>
+<li><a href="#listall">Listing all available public file spaces</a></li>
+<li><a href="#exsdss">Example: using the <strong>SDSS DR14</strong> public file service</a></li>
+</ul>
+
+</div>
+</div>
+</div>
+<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
+</div><div class="inner_cell">
+<div class="text_cell_render border-box-sizing rendered_html">
+<p><a class="anchor" id="summary"></a></p>
+<h1 id="Summary">Summary<a class="anchor-link" href="#Summary">&#182;</a></h1><p>Files in the virtual storage VOSpace are usually identified via the prefix "<em>vos://</em>". This shorthand identifier is resolved to a user's home directory of the storage space in the service.</p>
+<p>If the "<em>vos://</em>" prefix is instead the name of another user (e.g. "<em>geychaner://</em>"), and the remainder of the path grants public or group read/write access, then the other user's spaces will be accessed. Most user spaces have a "<em>/public</em>" directory to facilitate file sharing (e.g. '<em>geychaner://public/foo.fits</em>' will access the '<em>foo.fits</em>' file from user '<em>geychaner</em>'). Users can make any file (or directory) public by moving it to (or creating a link in) their "/public" directory.</p>
```

Comment:
I propose to use a different demo account here. Glenn Eychaner isn't working at DL anymore. May I suggest "demo00" as the standard strawperson user? I'll send you the password for it in Slack.